projects/demo/src/modules/components/dialog/examples/2/dialog-example/dialog-example.component.ts

```diff
@@ -36,12 +36,13 @@ import {injectContext} from '@taiga-ui/polymorpheus';
 })
 export class DialogExample {
     private readonly dialogs = inject(TuiDialogService);
-    private readonly context = injectContext<TuiDialogContext<number, number>>();
 
     protected value: number | null = null;
     protected name = '';
     protected items = [10, 50, 100];
 
+    public readonly context = injectContext<TuiDialogContext<number, number>>();
```

Comment:
Why access modifiers was changed ?

Reply:
We can only use public fields as context. Otherwise, we won't be able to determine the associated types.